### ISSUE TITLE
Ingress for Database

### ISSUE THREAD

**Issue by chrislovecnm** [cockroachdb/cockroach-operator#76]
The helm chart supports the creations of an ingress object.  Do we support this in the operator

```yaml
# CockroachDB's ingress for web ui.
ingress:
  enabled: false
  labels: {}
  annotations: {}
  #   kubernetes.io/ingress.class: nginx
  #   cert-manager.io/cluster-issuer: letsencrypt
  paths: [/]
  hosts: []
  # - cockroachlabs.com
  tls: []
  # - hosts: [cockroachlabs.com]
  #   secretName: cockroachlabs-tls
```

https://github.com/cockroachdb/helm-charts/blob/25e4598e13fa9402242d837e828a550a20e8a648/cockroachdb/values.yaml#L245

**Comment by chrislovecnm** [comment]
@johnrk this is a pretty big issue for "first issue", so I changed it to help wanted. We do need it, and after working on a couple of "first issues" this is a good issue to take on as you understand the code more.

**Comment by Szymongib** [comment]
Hi @chrislovecnm, 
Do I understand correctly, that the objective would be to create Ingress in Operator based on the configuration specified in the `CrdbCluster` CR? With config options mimicking the ones from the Helm chart?

If that is the case, would you mind if I take a stab at that?

**Comment by chrislovecnm** [comment]
@Szymongib i think our first iteration should be an internal elb.

we always appreciate help!

**Comment by pseudomuto** [comment]
Closed by #785
